orangecontrib/prototypes/widgets/owtwitternetwork.py

```diff
@@ -0,0 +1,90 @@
+import twitter
+import networkx as nx
+import orangecontrib.network as network
+
+from Orange.widgets import widget, gui
+from PyQt4 import QtGui
+
+class OWTwitterNetwork(widget.OWWidget):
+    name = "Twitter User Network"
+    description = ""
+    icon = "icons/Twitter.svg"
+    priority = 10
+    outputs = [("Followers", network.Graph), ("Following", network.Graph), ("All", network.Graph)]
+
+    want_main_area = False
+
+    def __init__(self):
+        super().__init__()
+
+        # GUI
+        box = gui.widgetBox(self.controlArea, "Info")
+        self.consumer_key = QtGui.QLineEdit()
+        box.layout().addWidget(QtGui.QLabel("Key:"))
+        box.layout().addWidget(self.consumer_key)
+        self.consumer_secret = QtGui.QLineEdit()
+        box.layout().addWidget(QtGui.QLabel("Secret:"))
+        box.layout().addWidget(self.consumer_secret)
+        box.layout().addWidget(QtGui.QLabel("Users:"))
+        self.users = QtGui.QTextEdit()
+        box.layout().addWidget(self.users)
+        self.infoa = gui.widgetLabel(box, '')
+        self.infob = gui.widgetLabel(box, '')
+        self.button = gui.button(box, self, "Create graph", self.fetch_users)
+
+    def fetch_users(self):
+        CONSUMER_KEY = self.consumer_key.text()
+        CONSUMER_SECRET = self.consumer_secret.text()
+        OAUTH_TOKEN = ''
+        OAUTH_TOKEN_SECRET = ''
+        auth = twitter.oauth.OAuth(OAUTH_TOKEN, OAUTH_TOKEN_SECRET, CONSUMER_KEY, CONSUMER_SECRET)
```

Comment:
It would be great to store credentials the way we do in Text add-on: https://github.com/biolab/orange3-text/blob/master/orangecontrib/text/credentials.py